packages/ponder-metadata-api/package.json

```diff
@@ -12,20 +12,25 @@
   "homepage": "https://github.com/namehash/ensnode/tree/main/packages/ponder-metadata-api",
   "keywords": ["Ponder"],
   "exports": {
+    ".": "./src/index.ts",
     "./middleware": "./src/middleware.ts"
   },
   "files": ["dist"],
   "publishConfig": {
     "access": "public",
     "exports": {
+      ".": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
```

Comment:
This package now also exports some types, so I created an root index.ts file to export those types, as well as the middleware function.